test/binCases/entry/multiple-config/multiple-config.test.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const { run, extractSummary } = require("../../../testUtils");
+
+test("multiple-config", () => {
+	const { stdout, code, stderr } = run(__dirname, [
+		"--config",
+		"./webpack.config.js",
+		"--output-filename",
+		"[name].js",
+		"--mode",
+		"production"
+	]);
+
+	const summary = extractSummary(stdout);
+
+	expect(code).toBe(0);
+	expect(code).toBeDefined();
+	expect(code).not.toBeNull();
+	expect(summary).toEqual(expect.anything());
```

Comment:
Bad idea to do it, no need test on anything

Reply:
@evilebottnawi should I remove it? :thinking: